examples/asr/asr_hybrid_transducer_ctc/helpers/convert_nemo_asr_hybrid_to_ctc.py

```diff
@@ -0,0 +1,170 @@
+    new_cfg['decoder'] = hybrid_model_cfg['aux_ctc']['decoder']
+    del new_cfg['compute_eval_loss']
+    del new_cfg['model_defaults']
+    del new_cfg['joint']
+    del new_cfg['decoding']
+    del new_cfg['aux_ctc']
+    del new_cfg['loss']
+    if BPE and 'labels' in new_cfg:
+        del new_cfg['labels']
+    elif (not BPE) and 'tokenizer' in new_cfg:
+        del new_cfg['tokenizer']
+    del new_cfg['target']
+    del new_cfg['nemo_version']
+
+    new_cfg_oc = OmegaConf.create(new_cfg)
+    ctc_model = ctc_class.restore_from(
+        args.input, map_location=torch.device('cpu'), override_config_path=new_cfg_oc, strict=False
+    )
+
+    assert all(
+        [
+            torch.allclose(hybrid_model.state_dict()[x], ctc_model.state_dict()[x])
+            for x in hybrid_model.state_dict().keys()
+            if x.split('.')[0] in ['preprocessor', 'encoder']
+        ]
+    ), "Encoder and preprocessor state dicts don't match!"
+
+    ctc_model.decoder.load_state_dict(hybrid_model.ctc_decoder.state_dict())
+
+    assert all(
+        [
+            torch.allclose(hybrid_model.ctc_decoder.state_dict()[x], ctc_model.decoder.state_dict()[x])
+            for x in hybrid_model.ctc_decoder.state_dict().keys()
+        ]
+    ), "Decoder state_dict load failed!"
+
+    assert isinstance(ctc_model, ctc_class), "Extracted CTC model is of the wrong expected class!"
+
+    return ctc_model
+
+
+def extract_model_rnnt(args, hybrid_model):
+    BPE = False
+    rnnt_class = EncDecRNNTModel
+    if 'tokenizer' in hybrid_model.cfg.keys():
+        BPE = True
+        rnnt_class = EncDecRNNTBPEModel
+
+    hybrid_model_cfg = OmegaConf.to_container(hybrid_model.cfg)
+
+    new_cfg = hybrid_model_cfg.copy()
+    del new_cfg['aux_ctc']
+    if BPE and 'labels' in new_cfg:
+        del new_cfg['labels']
+    elif (not BPE) and 'tokenizer' in new_cfg:
+        del new_cfg['tokenizer']
+    del new_cfg['target']
+    del new_cfg['nemo_version']
+
+    new_cfg_oc = OmegaConf.create(new_cfg)
+    rnnt_model = rnnt_class.restore_from(
+        args.input, map_location=torch.device('cpu'), override_config_path=new_cfg_oc, strict=False
+    )
+
+    assert all(
+        [
+            torch.allclose(hybrid_model.state_dict()[x], rnnt_model.state_dict()[x])
+            for x in hybrid_model.state_dict().keys()
+            if x.split('.')[0] in ['preprocessor', 'encoder', 'decoder', 'joint']
+        ]
+    ), "State dict values mismatch, something went wrong!"
+
+    assert isinstance(rnnt_model, rnnt_class), "Extracted RNNT model is of the wrong expected class!"
+
+    return rnnt_model
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--input', required=True, type=str, help='path to Nemo Hybrid model .nemo file')
+    parser.add_argument('-o', '--output', required=True, type=str, help='path and name of output .nemo file')
+    parser.add_argument(
+        '-m',
+        '--model',
```

Comment:
Probably should be called --type or --model_type

Reply:
Changed it to --model_type